charts/nd-common/templates/_istio.tpl

```diff
@@ -81,6 +76,25 @@ proxy.istio.io/overrides: >-
 
 {{/*
 
+Build a comma-separated list of ports to exclude from Istio routing.
+
+*/}}
+{{- define "nd-common.istioExcludedInboundPorts" -}}
+{{- $ports := list }}
+{{- with .Values.istio.excludeInboundPorts }}
+{{- range $port := index . }}
+{{- $portStr := $port | toString }}
+{{- $ports = tpl $portStr $ | append $ports }}
+{{- end }}
+{{- end }}
+{{- if and .Values.monitor.portNumber .Values.monitor.enabled }}
+{{- $ports = append $ports .Values.monitor.portNumber }}
+{{- end }}
+{{- join ", " $ports }}
```

Comment:
Is the space okay to have?

Reply:
it was in the previous template so just copied that 🤷‍♀️